style.css

```diff
@@ -53,6 +84,11 @@ li {
   font-size: 24px;
   font-weight: 700;
   padding: 6px;
+  transition: 0.7s ease;
+}
+
+.subheadline:hover {
+  color: #ff6b00;
 }
 
 .headlinetext {
```

Comment:
Great job @dgonzalesi 🤝

- [ ] Your text appears to be in italic style. You may please make your text appear the same as the one specified in the Figma template. 
- Your design
![Your text](https://user-images.githubusercontent.com/63917725/178076262-8f333bec-c045-4fb5-8b2c-4ffe818d7e3b.PNG)

- Figma design
![figma text](https://user-images.githubusercontent.com/63917725/178076312-4b69d2d4-cc83-4642-8989-010a0c4e72a3.PNG)